content/docs/philosophy/philosophy.md

```diff
@@ -0,0 +1,69 @@
+---
+title: Philosophy
+sort_rank: 2
+---
+
+# Do one thing well
+
+We believe in the [Unix
+philosophy](https://en.wikipedia.org/wiki/Unix_philosophy), abridged from [Doug
+McIlroy's initial version from 1978](http://emulator.pdp-11.org.ru/misc/1978.07_-_Bell_System_Technical_Journal.pdf):
+
+1. Make each program do one thing well.
+  While the scope of "one thing" invariably encompasses more and more
+  elements due to increased overall system and computing complexity, we are
+  still doing one thing: ingest metric data, do computations on it, and expose
+  it to other systems.
+2. Expect the output of every program to become the input to another, as yet unknown, program.
+  Today's lingua franca is HTTP endpoints, which are used by Prometheus
```

Comment:
HTTP endpoints with JSON

But this isn't what we use everywhere. We've our own text format, and file_sd and yaml for config files.

Reply:
That's why I didn't put JSON.

Reply:
But we're also doing many things that aren't HTTP.

Reply:
Is there something you would propose instead?

Reply:
I'd remove it.